ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -192,9 +192,15 @@ set_local_repo() {
     version="$(lsb_release -sr)"
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
+<<<<<<< HEAD
```

Comment:
(lint) Couldn't parse this here string. Fix to allow more checks. [SC1073]
(lint) The mentioned syntax error was in this simple command. [SC1009]
(lint) Fix any mentioned problems and try again. [SC1072]